qiita_db/test/test_analysis.py

```diff
@@ -95,12 +100,12 @@ def test_create(self):
 
         new = Analysis.create(User("admin@foo.bar"), "newAnalysis",
                               "A New Analysis")
-        self.assertEqual(new.id, 3)
+        self.assertEqual(new.id, 7)
```

Comment:
I can do it here, but it will also need to be added as something to be done as part of the DB/NODB breakup issue so it is consistent across all tests. If that's agreeable, I'll make the change.